### ISSUE TITLE
Freeplay starts with message referencing combat with "the natives"

### ISSUE THREAD

**Issue by dperelman** [Derpumu/Pacifist#62]
When you start a game, it shows the following message:
![image](https://github.com/Derpumu/Pacifist/assets/1740306/271d4008-0c5d-4a8d-a88a-fa038f7ecd88)

The last part about protecting yourself from the natives doesn't apply. Not sure if mods can change the locale strings for scenarios or if you would have to redefine the freeplay scenario.

(This doesn't actually affect how I plan to use the mod, since hosting multiplayer I'll be starting from a save just after this.)

**Comment by Derpumu** [comment]
just "overwriting" the locale cfg for freeplay scenario did not work.

**Comment by dperelman** [comment]
Yeah, as far as I can tell, mods can't modify scenarios from other mods, they can only replace them entirely. My workaround was to just manually edit the save file.

**Comment by Derpumu** [comment]
Nullius modifies this -> will hjave to investigate

**Comment by dperelman** [comment]
Disables freeplay intro here:
https://github.com/GregorSamsanite/nullius/blob/224f2a2a87d83760e0c133e38454d3b8b5f67b55/nullius/scripts/startup.lua#L185
```lua
    if (remote.interfaces["freeplay"] ~= nil) then
      remote.call("freeplay", "set_skip_intro", true)
```

Then displays its own intro here: https://github.com/GregorSamsanite/nullius/blob/224f2a2a87d83760e0c133e38454d3b8b5f67b55/nullius/scripts/startup.lua#L273
```lua
    if game.is_multiplayer() then
      player.print(intro)
    else
      game.show_message_dialog{text = intro}
    end
```
